osm2gtfs/osm2gtfs.py

```diff
@@ -65,8 +65,8 @@ def main():
     # Add data to feed
     agency_creator.add_agency_to_feed(feed)
     feed_info_creator.add_feed_info_to_feed(feed)
-    routes_creator.add_routes_to_feed(feed, data)
     stops_creator.add_stops_to_feed(feed, data)
+    routes_creator.add_routes_to_feed(feed, data)
```

Comment:
Why did you move stops before routes?

Reply:
I remember that it was necessary but not exactly why. Will report when reading through the code again while improving based on the other comments.

Reply:
Incofer and Fenix are using the stops already in the routes_creator. Without creating the stops properly, eventual overrides in custom creators, such as `_define_stop_id` would be executed too late the routes would have stops with wrong gtfs ids.